common/zoom/zoom-scheduled-meeting.ts

```diff
@@ -12,41 +12,36 @@ enum RecurrenceMeetingTypes {
     MONTHLY = 3,
 }
 
-type ZoomMeeting = {
+export type ZoomMeeting = {
+    agenda: string;
+    created_at: string;
+    duration: number;
+    host_id: string;
+    id: number;
+    join_url: string;
+    pmi: string;
+    start_time: string;
+    timezone: string;
+    topic: string;
+    type: number;
+    uuid: string;
+};
+
+export type ZoomMeetings = {
     next_page_token: string;
     page_count: number;
     page_number: number;
     page_size: number;
     total_records: number;
-    meetings: [
-        {
-            agenda: string;
-            created_at: string;
-            duration: number;
-            host_id: string;
-            id: number;
-            join_url: string;
-            pmi: string;
-            start_time: string;
-            timezone: string;
-            topic: string;
-            type: number;
-            uuid: string;
-        }
-    ];
+    meetings: ZoomMeeting[];
 };
 
 const zoomUsersUrl = 'https://api.zoom.us/v2/users';
 const zoomMeetingUrl = 'https://api.zoom.us/v2/meetings';
 const zoomMeetingReportUrl = 'https://api.zoom.us/v2/report/meetings';
 
-const createZoomMeeting = async (zoomUsers: ZoomUser[], startTime: Date, isCourse: boolean, endDateTime?: Date) => {
-    if (!isZoomFeatureActive()) {
-        // return test data if zoom is not active
-        return {
-            id: 123456789,
-        };
-    }
+const createZoomMeeting = async (zoomUsers: ZoomUser[], startTime: Date, isCourse: boolean, endDateTime?: Date): Promise<ZoomMeeting> => {
```

Comment:
This returns one single ZoomMeeting, correct? Or just it's idea? Either way, I'd prefer a type instead of any

Reply:
Yeah, it's way better like this.